OpenRA.Mods.Common/Traits/Air/AttackPlane.cs

```diff
@@ -37,7 +37,7 @@ public override Activity GetAttackActivity(Actor self, Target newTarget, bool al
 			return new FlyAttack(self, newTarget);
 		}
 
-		protected override bool CanAttack(Actor self, Target target)
+		public override bool CanAttack(Actor self, Target target)
```

Comment:
This only seems to be necessary because you split out the internal activities. Why?